Resources/config/services.yml

```diff
@@ -14,6 +14,8 @@ parameters:
   lch.media.pdf.regenerate_thumbnails.command.class: Lch\MediaBundle\Command\RegeneratePdfThumbnailsCommand
   lch.media.pdf.subscriber.class: Lch\MediaBundle\Listener\PdfSubscriber
 
+  lch.media.behavior.reflection.class_analyzer.class: Lch\MediaBundle\Behavior\ClassAnalyzer
```

Comment:
The `Behavior` folder should only traits, no classes. Could you move it away please?

Reply:
Ok, i will change it